extensions/functions_arithmetic.yaml

```diff
@@ -175,12 +175,20 @@ scalar_functions:
         return: fp64
   -
     name: "divide"
-    description: "Divide one value by another. Partial values are truncated."
+    description: >
+      Divide x by y. In the case of integer division, partial values are truncated.
+      The `on_division_by_zero` option governs behaviour in cases where y is 0 and x is not 0.
```

Comment:
Let's stick to American English for the documentation for consistency (behaviour -> behavior).

Reply:
Oops, updated now!